src/components/data/DataTable.js

```diff
@@ -215,6 +215,10 @@ const DataTable = (props) => {
   const getAllEntities = async () => {
     const params = _.pickBy(_.trim, { pageSize: filteredCount, filterTerms: activeTextFilter, filterOperator });
     const queryResults = await Ajax(signal).Workspaces.workspace(namespace, name).paginatedEntitiesOfType(entityType, params);
+    Ajax().Metrics.captureEvent(Events.workspaceDataFullTableSearch, {
+      workspaceNamespace: namespace, workspaceName: name, providerName: dataProvider.providerName,
+      cloudPlatform: dataProvider.providerName === wdsProviderName ? cloudProviders.azure.label : cloudProviders.gcp.label
+    })
```

Comment:
There's an [`extractWorkspaceDetails`](https://github.com/DataBiosphere/terra-ui/blob/91531a2da3bcce9128900ad8ed43753c74583a17/src/libs/events.js#L125-L140) convenience function for getting the common properties passed to `captureEvent`.

```suggestion
    Ajax().Metrics.captureEvent(Events.workspaceDataFullTableSearch, {
      ...extractWorkspaceDetails(workspace),
      providerName: dataProvider.providerName,
    })
```